### ISSUE TITLE
MariaDB replica process not updated

### ISSUE THREAD

**Issue by SalvoLunar** [flynn/flynn#4515]
Hi,

This error has happened a few times.
The MariaDB process crashes and the only replica left is days old resulting in my websites suddenly showing old content.

Any idea how to fix or at least monitor this issue?

**Comment by SalvoLunar** [comment]
I'm having a similar issue again... I've checked my mariadDB logs this morning and I've found this:

`[2019-04-15T21:39:29.795097Z]    2019-04-15 21:39:29 140712106015488 [ERROR] Slave SQL: Error during XID COMMIT: failed to update GTID state in mysql.gtid_slave_pos: 1062: Duplicate entry '0-71143' for key 'PRIMARY', Gtid 0-1684298499-71198, Internal MariaDB error code: 1942
[2019-04-15T21:39:29.8021Z]      2019-04-15 21:39:29 140712106015488 [ERROR] Slave (additional info): Duplicate entry '0-71143' for key 'PRIMARY' Error_code: 1062
[2019-04-15T21:39:29.802279Z]    2019-04-15 21:39:29 140712106015488 [Warning] Slave: Duplicate entry '0-71143' for key 'PRIMARY' Error_code: 1062
[2019-04-15T21:39:29.802424Z]    2019-04-15 21:39:29 140712106015488 [ERROR] Error running query, slave SQL thread aborted. Fix the problem, and restart the slave SQL thread with "SLAVE START". We stopped at log 'mariadb-bin.000005' position 31652764; GTID position '0-1684298499-71197'
[2019-04-15T21:39:29.802565Z]    2019-04-15 21:39:29 140712106015488 [Note] Slave SQL thread exiting, replication stopped in log 'mariadb-bin.000005' at position 31652764; GTID position '0-1684298499-71197'`

Does anyone have any suggestion on how to fix this without having to rebuild the whole cluster?
